src/index.html

```diff
@@ -5,10 +5,40 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+        src="moyo_1.png"
+        alt="MoyoLogo"
+        class="logo__img">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="index.html" class="nav__link is-active">APPLE</a></li>
+          <li class="nav__item"><a href="index.html" class="nav__link">SAMSUNG</a></li>
+          <li class="nav__item"><a href="index.html" class="nav__link">SMARTPHONES</a></li>
+          <li class="nav__item">
+            <a
+            href="index.html"
+            class="nav__link"
+            data-qa="hover"
+            >
+            LAPTOPS & COMPUTERS
+            </a>
+          </li>
```

Comment:
code it not aligned, check again the checklist 
```suggestion
            <a
              href="index.html"
              class="nav__link"
              data-qa="hover"
            >
              LAPTOPS & COMPUTERS
            </a>
          </li>
```